### PR TITLE
Multilingual dataset wrapper and multilingual MS-COCO captions

### PR DESCRIPTION
**Pull request by FreddeFrallan**
A wrapper for dataset class, that maps the original captions into their corresponding cross-language captions.
These translated captions are stored in the form of a JSON file.

Also implemented automatic download and support for the XTD-Dataset (multilingual MS-COCO)

### CODE REVIEW

**Comment by rom1504** [comment]
could you fix the test https://github.com/LAION-AI/CLIP_benchmark/actions/runs/3503991588/jobs/5869390121 ?

File: clip_benchmark/datasets/multilingual_mscoco-old.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+from subprocess import call
```
Comment (rom1504) [inline clip_benchmark/datasets/multilingual_mscoco-old.py:1]: why a -old and not old ?

File: clip_benchmark/datasets/builder.py [hunk 2]
```diff
@@ -158,18 +168,42 @@ def build_dataset(dataset_name, root="root", transform=None, split="test", downl
...
+        root_split = os.path.join(root, get_archive_name(split).replace(".zip", ""))
+        if (language != 'en'):
+            # Multilingual MS-COCO or XTD-Dataset
+            # 
+            # If we are loading any multilingual version we override the annotation file
+            # We also override the root_split, since the multilingual_en has images from all splits
+            # Instead the image paths have been altered to include to split-folders
+            # We therefore also check so that all splits have been downloaded
+            from clip_benchmark.datasets import multilingual_mscoco
+            annotation_file = os.path.join(root, multilingual_mscoco.IMAGE_INDEX_FILE)
```
Comment (rom1504) [inline clip_benchmark/datasets/builder.py:198]: it feels like coco and multilingual coco are 2 different datasets
can we simply consider them as such ?

File: clip_benchmark/datasets/multilingual_mscoco-old.py (hunk 1 above)
Comment (FreddeFrallan) [reply clip_benchmark/datasets/multilingual_mscoco-old.py]: Woops, this filde should not have been included. It has now been removed from the commit.

File: clip_benchmark/datasets/builder.py (hunk 2 above)
Comment (FreddeFrallan) [reply clip_benchmark/datasets/builder.py]: Very good point! I have now implemented it as its own dataset.

**Comment by rom1504** [comment]
looking good now. I'll try it out and then I think we can merge

**Comment by rom1504** [comment]
works, let's merge

interesting follow up:
* generic clip support to eval mclip
* script similar to run.sh to run all multilingual evals (maybe we simply make a dataset.csv with a language column)

**Pull request merged**